Sources/Views/EpisodePage.swift

```diff
@@ -848,10 +850,9 @@ public let useCreditCTA = "Use an episode credit"
 
 let divider = Node.hr(attributes: [.class([Class.pf.components.divider])])
 
-func transcriptView(
+private func transcriptView(
```

Comment:
can make a few things in here private now that we got rid of the old episode page